README.md

```diff
@@ -149,6 +149,7 @@ package { 'xPSDesiredStateConfiguration':
   ensure   => latest,
   provider => 'windowspowershell',
   source   => 'PSGallery',
+  install_options => [ '-AllowClobber' ]
```

Comment:
Could you add an example showing how to pass parameters as well as switches using the hash syntax?